nemo/collections/tts/parts/waveglow.py

```diff
@@ -85,17 +131,16 @@ def __init__(self, n_in_channels, n_mel_channels, n_layers, n_channels, kernel_s
         end.bias.data.zero_()
         self.end = end
 
+        cond_layer = torch.nn.Conv1d(n_mel_channels, 2 * n_channels * n_layers, 1)
+        self.cond_layer = torch.nn.utils.weight_norm(cond_layer, name='weight')
+
         for i in range(n_layers):
             dilation = 2 ** i
             padding = int((kernel_size * dilation - dilation) / 2)
             in_layer = torch.nn.Conv1d(n_channels, 2 * n_channels, kernel_size, dilation=dilation, padding=padding,)
             in_layer = torch.nn.utils.weight_norm(in_layer, name='weight')
             self.in_layers.append(in_layer)
 
-            cond_layer = torch.nn.Conv1d(n_mel_channels, 2 * n_channels, 1)
```

Comment:
I do not understand this change. You moved this out before the loop? Before, number of "cond_layers" was equal to n_layers, and now it is only one?